src/popup/components/dashboard/Card.vue

```diff
@@ -1,31 +1,53 @@
 <template>
-  <div
-    class="card"
-    :class="{ 'is-big': isBig, clickable, dense }"
-    :style="styleComponent"
-    @click="$emit('click', $event)"
-  >
-    <div class="card-icon">
-      <slot
-        name="icon"
-        class="icon-svg"
-      />
-    </div>
-    <div>
-      <div class="title">
-        {{ title }}
+  <transition name="fade">
+    <div
+      v-if="!cardId || !hiddenCards || !hiddenCards.includes(cardId)"
```

Comment:
I don't see a reason for hiding the card if there is no `cardId` and if there are no `hiddenCards`.
Also I suggest to put this kind of logic under `isVisible` computed the debugging easier.